packages/app/src/components/collapsible/collapsible-section.tsx

```diff
@@ -115,8 +115,18 @@ export const CollapsibleSection = ({
    * If so, the collapsible needs to be opened.
    */
   useEffect(() => {
-    const isOpenedByQueryParam = window.location.hash.substr(1) === id;
-    setOpen(isOpenedByQueryParam);
+    const onHashChange = function () {
```

Comment:
We use this pattern of `onEvenName`/`handleEventName`, where the `on` is used for prop names, and `handle` is used for the function actually handling the event. It makes things more clear especially when you need to locally handle the event and then call another handler from the props.

Also we prefer named functions over unnamed lambdas and usually only use lambdas inline

 ```suggestion
    function handleHashChange()  {
```